mso/resource_mso_schema_site.go

```diff
@@ -213,13 +213,14 @@ func resourceMSOSchemaSiteDelete(d *schema.ResourceData, m interface{}) error {
 			log.Printf("[DEBUG] Parse of JSON failed with err: %s.", err)
 			return err
 		}
-		req, err := msoClient.MakeRestRequest("POST", "mso/api/v1/task", payload, true)
+		req, err := msoClient.MakeRestRequest("POST", "api/v1/task", payload, true)
 		if err != nil {
 			log.Printf("[DEBUG] MakeRestRequest failed with err: %s.", err)
 			return err
 		}
-		_, _, err = msoClient.Do(req)
-		if err != nil {
+		_, resp, err := msoClient.Do(req)
+		if err != nil || resp.StatusCode != 202 {
+			log.Printf("[DEBUG] Request failed with resp: %v.", resp)
 			log.Printf("[DEBUG] Request failed with err: %s.", err)
```

Comment:
Why two log statements and not combined them?

Reply:
no reason, combined now, also in the deploy resource